### ISSUE TITLE
Text Sets Keyboard Navigation

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#5878]
## Bug Description

Noticed while testing out missing keyboard functionality in page layouts that text sets also needs to get keyboard functionality added. 

## Expected Behaviour

Should be able to tab into text sets and then use my arrow keys to navigate them and tab again to exit text sets. See media keyboard functionality. 

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
I think this will be carried over to the next sprint because it relies on #5876 for completion. This was an internally discovered bug though so I feel like it's an ok thing to just carry over to the next release.

**Comment by csossi** [comment]
Verified in QA
